syncthing.fc

```diff
@@ -0,0 +1,8 @@
+# binary
+/usr/bin/syncthing                      -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+/usr/local/bin/syncthing                -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+HOME_DIR/.local/bin/syncthing           -- gen_context(system_u:object_r:syncthing_exec_t,s0)
```

Comment:
There are multiple ways of installing syncthing. I could probably condense `/usr/(local/)?bin/syncthing`.

Installing Syncthing as a fat binary in your homedir allows Syncthing to automatically update itself outside of package management. While that's a whole different problem, some users might want this for distros that aren't yet packaging it (ie Fedora).